app/src/views/DrefApplicationForm/schema.ts

```diff
@@ -731,7 +751,70 @@ const schema: DrefFormSchema = {
                         people_targeted_with_early_actions: {
                             validations: [positiveIntegerCondition],
                         },
+                        proposed_action: {
+                            keySelector: (n) => n.client_id,
+                            member: () => ({
+                                fields: (): ProposedActionsFields => ({
+                                    client_id: {},
```

Comment:
This should be required as per my knowledge. Can you confirm @samshara @tnagorra @frozenhelium ?